### PR TITLE
[TRANSLATION] Update Russian (ru-RU) translation

### PR DESCRIPTION
**Pull request by diakin**
## Update russian (ru-RU) translation

_Do a quick recap of your work here._

JIRA issue: [CORE-XXXX](https://jira.reactos.org/browse/CORE-XXXX)

## Proposed changes

_Describe what you propose to change/add/fix with this pull request._

- 
- 

## TODO

_Use a TODO when your pull request is Work in Progress._

- [ ] 
- [ ]

### CODE REVIEW

File: base/applications/calc/lang/ru-RU.rc [hunk 1]
```diff
@@ -429,8 +429,8 @@ END
 /* Angles */
 STRINGTABLE
 BEGIN
-    IDS_ANGLE_ARCMINUTES "Minute of arc"
-    IDS_ANGLE_ARCSECONDS "Second of arc"
+    IDS_ANGLE_ARCMINUTES "Угловые минуты"
+    IDS_ANGLE_ARCSECONDS "Угловые секунды"
```
Comment (oleg-dubinskiy) [inline base/applications/calc/lang/ru-RU.rc:433]: В английском переводе используется единственное число, а не множественное. Поэтому в русском более логично также.
In English translation, singular is used here, so it's more logical to do the same for Russian also.
```suggestion
    IDS_ANGLE_ARCMINUTES "Угловая минута"
    IDS_ANGLE_ARCSECONDS "Угловая секунда"
```

File: base/applications/cmdutils/at/lang/ru-RU.rc [hunk 2]
```diff
@@ -31,7 +31,7 @@ id                 Порядковый номер запланированно
     IDS_CONFIRM_YES "Y"
     IDS_CONFIRM_NO "N"
 
-    IDS_DELETE_ALL "This operation will delete all scheduled jobs.\n"
+    IDS_DELETE_ALL "Эта операция удалит все запланированные задачи\n"
```
Comment (oleg-dubinskiy) [inline base/applications/cmdutils/at/lang/ru-RU.rc:34]: Дополнительный пробел в конце не обязателен.
The extra whitespace at the end0 is not necessary.
```suggestion
    IDS_DELETE_ALL "Эта операция удалит все запланированные задачи\n"
```

File: base/applications/fltmc/lang/ru-RU.rc [hunk 3]
```diff
@@ -11,7 +11,7 @@ BEGIN
     IDS_USAGE_LOAD      "Загружает драйвер фильтра\n"
     IDS_USAGE_UNLOAD    "Выгружает драйвер фильтра\n"
     IDS_USAGE_FILTERS   "Выводит список всех зарегистрированных фильтров\n"
-    IDS_USAGE_VOLUMES   "Lists all volumes\n"
+    IDS_USAGE_VOLUMES   "Перечисляет все тома\n"
```
Comment (oleg-dubinskiy) [inline base/applications/fltmc/lang/ru-RU.rc:14]: В контексте предназначения использования этого параметра, здесь больше подходит слово "Отображает".
Basing on the purpose of this command parameter, "Displaying" looks more suitable here.
```suggestion
    IDS_USAGE_VOLUMES   "Отображает все тома\n"
```

File: base/applications/fltmc/lang/ru-RU.rc [hunk 4]
```diff
@@ -27,5 +27,5 @@ BEGIN
     IDS_ERROR_FILTERS   "Не удалось получить список фильтров (0x%X)\n"
     IDS_ERROR_LOAD      "Не удалось загрузить фильтр (0x%X)\n"
     IDS_ERROR_UNLOAD    "Не удалось выгрузить фильтр (0x%X)\n"
-    IDS_ERROR_VOLUMES   "Failed to list the volumes (0x%X)\n"
+    IDS_ERROR_VOLUMES   "Не удалось перечислить тома (0x%X)\n"
```
Comment (oleg-dubinskiy) [inline base/applications/fltmc/lang/ru-RU.rc:30]: Такое же замечание и здесь.
The same comment here.
```suggestion
    IDS_ERROR_VOLUMES   "Не удалось отобразить тома (0x%X)\n"
```

**Review by oleg-dubinskiy — COMMENTED** [review]
В остальном замечаний нет.
In everything other, LGTM.
:slightly_smiling_face:

File: base/applications/cmdutils/at/lang/ru-RU.rc (hunk 2 above)
Comment (diakin) [reply base/applications/cmdutils/at/lang/ru-RU.rc:34]: Там кстати точка в конце английского. Добавить? 
There is dot in English translation, maybe add it in Russian too?

Comment (oleg-dubinskiy) [reply base/applications/cmdutils/at/lang/ru-RU.rc]: Да, правильно.
Yes, correct.

File: base/applications/cmdutils/at/lang/ru-RU.rc [hunk 5]
```diff
@@ -31,7 +31,7 @@ id                 Порядковый номер запланированно
...
```
Comment (oleg-dubinskiy) [inline base/applications/cmdutils/at/lang/ru-RU.rc:34]: Всё ещё забыли добавить точкув конце.
Still fotgot to add a dot at the end.
```suggestion
    IDS_DELETE_ALL "Эта операция удалит все запланированные задачи.\n"
```

**Review by oleg-dubinskiy — APPROVED** [review]

File: base/applications/fltmc/lang/ru-RU.rc (hunk 3 above)
Comment (diakin) [reply base/applications/fltmc/lang/ru-RU.rc]: "Отображает все тома\n"  -  This suggestion has been applied. Is it means that all is OK and I don't need to do anything?

Comment (oleg-dubinskiy) [reply base/applications/fltmc/lang/ru-RU.rc]: Ah, found one more suggestion to improve. :smirk:

File: base/applications/cmdutils/find/lang/ru-RU.rc [hunk 6]
```diff
@@ -8,11 +8,11 @@ FIND [/V] [/C] [/N] [/I] [/OFF[LINE]] ""строка"" [[диск:][путь]и
...
   /I         Поиск без учета регистра символов.\n\
-  /OFF[LINE] Do not skip files that have the offline attribute set.\n\
+  /OFF[LINE] Не пропускать файлы, у которых установлен атрибут offline.\n\
   ""строка""   Искомая строка.\n\
   [диск:][путь]имя_файла\n\
              Один или несколько файлов, в которых выполняется поиск."
     IDS_NO_SUCH_FILE "FIND: %s: Файл не существует.\n"
     IDS_CANNOT_OPEN "FIND: %s: Невозможно открыть файл.\n"
-    IDS_INVALID_SWITCH "FIND: Invalid switch\n"
+    IDS_INVALID_SWITCH "FIND: Неправильный ключ\n"
```
Comment (oleg-dubinskiy) [inline base/applications/cmdutils/find/lang/ru-RU.rc:17]: In this context, "switch" or "key" means a keyword for searching the file. So it's looks better as follows:
```suggestion
    IDS_INVALID_SWITCH "FIND: Неправильный поисковый запрос\n"
```
The "key" might also mean "Activation Key", "Product Key", so it might cause some misunderstanding. :slightly_smiling_face:

File: base/applications/fltmc/lang/ru-RU.rc (hunk 3 above)
Comment (diakin) [reply base/applications/fltmc/lang/ru-RU.rc]: Fixed! )

File: base/applications/cmdutils/find/lang/ru-RU.rc [hunk 7]
```diff
@@ -8,11 +8,11 @@ FIND [/V] [/C] [/N] [/I] [/OFF[LINE]] ""строка"" [[диск:][путь]и
...
   /I         Поиск без учета регистра символов.\n\
-  /OFF[LINE] Do not skip files that have the offline attribute set.\n\
+  /OFF[LINE] Не пропускать файлы, у которых установлен атрибут offline.\n\
   ""строка""   Искомая строка.\n\
   [диск:][путь]имя_файла\n\
              Один или несколько файлов, в которых выполняется поиск."
     IDS_NO_SUCH_FILE "FIND: %s: Файл не существует.\n"
     IDS_CANNOT_OPEN "FIND: %s: Невозможно открыть файл.\n"
-    IDS_INVALID_SWITCH "FIND: Invalid switch\n"
+    IDS_INVALID_SWITCH "FIND: Неправильный поисковый запрос\n"
```
Comment (binarymaster) [inline base/applications/cmdutils/find/lang/ru-RU.rc:17]:
```suggestion
    IDS_INVALID_SWITCH "FIND: Неправильный ключ\n"
```

File: base/applications/cmdutils/xcopy/lang/ru-RU.rc [hunk 8]
```diff
@@ -52,7 +52,7 @@ XCOPY источник [целевой_объект] [/I] [/S] [/Q] [/F] [/L] [/
 /A  Копирует только те файлы, для которых установлен атрибут ""архивный"".\n\
 /M  Копирует только те файлы, для которых установлен атрибут ""архивный"",\n\
       при этом атрибут удаляется.\n\
-[/K]  Copy file attributes, without this attributes are not preserved.\n\
+[/K]  Копирует атрибуты файла, без этого атрибуты не сохраняются.\n\
```
Comment (binarymaster) [inline base/applications/cmdutils/xcopy/lang/ru-RU.rc:55]:
```suggestion
[/K]  Копирует атрибуты файлов, без этого атрибуты не сохраняются.\n\
```

File: base/applications/fltmc/lang/ru-RU.rc [hunk 9]
```diff
@@ -11,7 +11,7 @@ BEGIN
     IDS_USAGE_LOAD      "Загружает драйвер фильтра\n"
     IDS_USAGE_UNLOAD    "Выгружает драйвер фильтра\n"
     IDS_USAGE_FILTERS   "Выводит список всех зарегистрированных фильтров\n"
-    IDS_USAGE_VOLUMES   "Lists all volumes\n"
+    IDS_USAGE_VOLUMES   "Отображает все тома\n"
```
Comment (binarymaster) [inline base/applications/fltmc/lang/ru-RU.rc:14]:
```suggestion
    IDS_USAGE_VOLUMES   "Выводит список всех томов\n"
```

File: base/applications/fltmc/lang/ru-RU.rc [hunk 10]
```diff
@@ -27,5 +27,5 @@ BEGIN
     IDS_ERROR_FILTERS   "Не удалось получить список фильтров (0x%X)\n"
     IDS_ERROR_LOAD      "Не удалось загрузить фильтр (0x%X)\n"
     IDS_ERROR_UNLOAD    "Не удалось выгрузить фильтр (0x%X)\n"
-    IDS_ERROR_VOLUMES   "Failed to list the volumes (0x%X)\n"
+    IDS_ERROR_VOLUMES   "Не удалось отобразить тома (0x%X)\n"
```
Comment (binarymaster) [inline base/applications/fltmc/lang/ru-RU.rc:30]:
```suggestion
    IDS_ERROR_VOLUMES   "Не удалось получить список томов (0x%X)\n"
```

File: ntoskrnl/include/internal/i386/mm.h [hunk 11]
```diff
@@ -163,7 +163,7 @@ C_ASSERT(PD_COUNT == 1);
 
 #define MI_SYSTEM_PTE_BASE (PVOID)MiAddressToPte(NULL)
 
-/* Base addreses for page directories */
+/* Base addresses for page directories */
```
Comment (binarymaster) [inline ntoskrnl/include/internal/i386/mm.h:166]: Unrelated change outside translations.

File: ntoskrnl/include/internal/ke_x.h [hunk 12]
```diff
@@ -996,7 +996,7 @@ KiComputeDueTime(IN PKTIMER Timer,
 
 //
 // Called from Unlink and Queue Insert Code.
-// Also called by timer code when canceling an inserted timer.
+// Also called by timer code when cancelling an inserted timer.
```
Comment (binarymaster) [inline ntoskrnl/include/internal/ke_x.h:999]: Unrelated changes outside translations (and above).

File: base/applications/cmdutils/find/lang/ru-RU.rc (hunk 7 above)
Comment (binarymaster) [reply base/applications/cmdutils/find/lang/ru-RU.rc:17]: > In this context, "switch" or "key" means a keyword for searching the file.

@oleg-dubinskiy no, it's for the unrecognized command-line switch.

File: ntoskrnl/include/internal/i386/mm.h (hunk 11 above)
Comment (diakin) [reply ntoskrnl/include/internal/i386/mm.h:166]: It's not translation, it's previous spell checking corrections.

File: ntoskrnl/include/internal/ke_x.h (hunk 12 above)
Comment (diakin) [reply ntoskrnl/include/internal/ke_x.h:999]: It's not translation, it's previous  PR  to spell checking corrections.

Comment (diakin) [reply ntoskrnl/include/internal/ke_x.h]: Fixed

File: base/applications/cmdutils/find/lang/ru-RU.rc (hunk 7 above)
Comment (oleg-dubinskiy) [reply base/applications/cmdutils/find/lang/ru-RU.rc]: Oops, okay, my mistake. :sweat_smile:

File: base/applications/cmdutils/find/lang/ru-RU.rc (hunk 6 above)
Comment (binarymaster) [inline base/applications/cmdutils/find/lang/ru-RU.rc:17]:
```suggestion
    IDS_INVALID_SWITCH "FIND: Недопустимый ключ\n"
```
(to keep it the same as in FC's `IDS_INVALID_SWITCH`)

**Review by binarymaster — APPROVED** [review]

**Comment by binarymaster** [comment]
> Do I need to do something else with this pull request or are there no more questions about it? I haven’t figured out how it all works in Github yet.

I think it's ready for merge, currently just waiting for CI to complete.

> Where can I discuss questions about Github? Is there any forum?

Use chat: https://chat.reactos.org (and Russian channel specifically: https://chat.reactos.org/reactos/channels/russian)
